cmd/main.go

```diff
@@ -436,8 +436,11 @@ func uploadBaseISOWithLeader(uploadLeader leader.ElectorInterface, objectHandler
 	}
 	return uploadLeader.RunWithLeader(ctx, func() error {
 		log.Info("Starting base ISO upload")
-		err = generator.UploadBaseISO()
-		log.Info("Finished base ISO upload")
-		return err
+		resp, err := http.Get(s3wrapper.RHCOSBaseURL)
```

Comment:
Why use a closure and not execute the logic here?
It seems that the code calling this function might log an error "Failed uploading base ISO" before the upload logic is executed.
https://sourcegraph.com/github.com/openshift/assisted-service@66ab7e7ad61f528b74fda9299538edabee6439b9/-/blob/cmd/main.go#L350

Reply:
I don't think I addressed this. I'll have another look at this tomorrow.